src/spriteSheet.js

```diff
@@ -142,7 +149,7 @@
       properties = properties || {};
 
       if (!kontra._isImage(properties.image)) {
-        throw Erorr('You must provide an Image for the SpriteSheet');
+        throw new Error('You must provide an Image for the SpriteSheet');
```

Comment:
The `new` keyword [isn't needed](https://github.com/jed/140bytes/wiki/Byte-saving-techniques#omit-the-new-keyword-when-possible) (and helps save bytes)